src/lib/import-export/export/export-database.ts

```diff
@@ -18,13 +18,19 @@ export async function exportDatabaseToFile(
 	const tempFilePath = path.join( site.path, tempFileName );
 
 	// Execute the command to export directly to the temp file
-	const { stderr } = await server.executeWpCliCommand( `db export ${ tempFileName }` );
+	const { stderr, exitCode } = await server.executeWpCliCommand(
+		`sqlite export ${ tempFileName } --require=/tmp/sqlite-command/command.php`
+	);
 
 	if ( stderr ) {
 		console.error( 'Error during export:', stderr );
 		throw new Error( `Database export failed: ${ stderr }` );
 	}
 
+	if ( exitCode ) {
+		throw new Error( 'Database import failed' );
```

Comment:
I'm updating the message to reference export as this file is part of the export process:
```suggestion
		throw new Error( 'Database export failed' );
```